src/Cli/dotnet/commands/dotnet-test/VSTestForwardingApp.cs

```diff
@@ -19,7 +21,10 @@ public VSTestForwardingApp(IEnumerable<string> argsToForward)
             if (!hasRootVariable)
             {
                 WithEnvironmentVariable(rootVariableName, rootValue);
+                VSTestTrace.WriteTrace($"Root variable set {rootVariableName}:{rootValue}");
             }
+
+            VSTestTrace.SafeWriteTrace(() => $"Forwarding to '{GetVSTestExePath()}' with args \"{argsToForward?.Aggregate((a, b) => $"{a} | {b}")}\"");
```

Comment:
I don't know if that matters but in the other similar trace you did `{a}|{b}` (no space around `|`).

Reply:
I'll add space everywhere thx